### PR TITLE
Addressing sunpy/irispy/#107

### PR DESCRIPTION
**Pull request by byrdie**
-Added paragraph explaining how to extract data as numpy arrays from `NDCubeSequence` objects.

### CODE REVIEW

File: docs/ndcubesequence.rst [hunk 1]
```diff
@@ -375,3 +375,42 @@ dimensions.shape ``(<Quantity 3.0 pix>, <Quantity 3.0 pix>, <Quantity
...
+Extracting Data Arrays
+-------------------------
+
+It is possible that you may have some procedures that are designed to operate on arrays instead of
+`~ndcube.NDCubeSequence` objects.
+Therefore, it may be useful to extract the `data` (or any other information such as `mask`) in the `~ndcube.NDCubeSequence`
+into a single `~numpy.ndarray`.
+A succinct way of doing this operation is using python's list comprehension features.
+
+In the above examples we defined the `my_sequence` `~ndcube.NDCubeSequence` obejct.::
```
Comment (DanRyanIrish) [inline docs/ndcubesequence.rst]: "object" mispelled.

File: docs/ndcubesequence.rst [hunk 2]
```diff
@@ -375,3 +375,42 @@ dimensions.shape ``(<Quantity 3.0 pix>, <Quantity 3.0 pix>, <Quantity
...
+
+    >>> # Print dimensions of my_sequence as a reminder
+    >>> print(my_sequence.dimensions)
+    (<Quantity 3. pix>, <Quantity 3. pix>, <Quantity 4. pix>, <Quantity 5. pix>)
+
+In this section we will use this object to demonstrate extracting data arrays from `~ndcube.NDCubeSequence` objects.
+For example, say we wanted to make a 4D array out of `my_sequence`.::
```
Comment (DanRyanIrish) [inline docs/ndcubesequence.rst]: Suggested change: "...make a 4D array out of the data arrays within the `~ndcube.NDCube`s of `my_sequence`."

**Comment by DanRyanIrish** [comment]
Hi @byrdie.  Some final tiny changes and then I think it'll be ready :)

**Comment by DanRyanIrish** [comment]
👍

**Comment by DanRyanIrish** [comment]
Congratulations @byrdie on your first IRISpy merged PR!  Thanks for your contribution!

**Comment by DanRyanIrish** [comment]
Sorry!  I of course meant ndcube!

**Comment by byrdie** [comment]
Thanks! Nice to be part of the team.

**Pull request merged**